modules/core/src/channels.cpp

```diff
@@ -341,8 +341,8 @@ namespace cv
 {
 static bool ipp_extractChannel(const Mat &src, Mat &dst, int channel)
 {
-#ifdef HAVE_IPP_IW
-    CV_INSTRUMENT_REGION_IPP();
+#ifdef HAVE_IPP_IW_LL
+    CV_INSTRUMENT_REGION_IPP()
```

Comment:
Why is ';' ending removed?

MSVS editor doesn't like that.

Reply:
Perhaps it was done unintentionally, returned semicolon back.